tardis/io/decay.py

```diff
@@ -86,4 +86,37 @@ def decay(self, t):
         
         df = IsotopeAbundances.from_materials(decayed_materials)
         df.sort_index(inplace=True)
-        return df 
\ No newline at end of file
+        return df 
+
+    def merge_isotopes(self):
+        """
+        Merge Isotope dataframe according to atomic number 
+
+        Returns:
+            : merged isotope abundances
+        """
+
+        return self.groupby('atomic_number').sum()
+
+    def as_atomic_numbers(self, abundance, normalize=True):
```

Comment:
I'd call this method something like `merge_with` or only `merge` as that reflects what this function does, merge Isotopes with a normal DataFrame.
Maybe we can call the other argument `other` so we can easily distinguish between `self` and `other`.